source/MaterialXCore/Document.cpp

```diff
@@ -883,6 +883,55 @@ void Document::upgradeVersion(int desiredMajorVersion, int desiredMinorVersion)
     {
         // Convert material Elements to Nodes
         convertMaterialsToNodes(true);
+
+        // Update atan2 interface
+        const string ATAN2 = "atan2";
+        const string IN1 = "in1";
+        const string IN2 = "in2";
+
+        // Update nodedefs
+        for (auto nodedef : getMatchingNodeDefs(ATAN2))
```

Comment:
To avoid changes which would make stdlib_defs.mtlx incompatible with 1.37, the nodedef upgrade is done here.